bitnami/etcd/3.5/debian-12/rootfs/opt/bitnami/scripts/libetcd.sh

```diff
@@ -447,7 +447,7 @@ setup_etcd_active_endpoints() {
         ETCD_ACTIVE_ENDPOINTS=$(echo "${active_endpoints_array[*]}" | tr ' ' ',')
         export ETCD_ACTIVE_ENDPOINTS
     fi
-    echo "${active_endpoints} ${cluster_size} ${ETCD_ACTIVE_ENDPOINTS}"
+    echo "${active_endpoints} ${cluster_size} ${ETCD_ACTIVE_ENDPOINTS} ${host} ${port}"
```

Comment:
Can you adapt the docs to the new returns?

https://github.com/bitnami/containers/blob/d22de96483bb3035a7b0fdaecbf9dba97091a745/bitnami/etcd/3.5/debian-12/rootfs/opt/bitnami/scripts/libetcd.sh#L418-L424